tket/include/tket/PauliGraphRefactor/PGBox.hpp

```diff
@@ -83,6 +85,70 @@ class PGBox : public PGOp {
   std::vector<SpPauliStabiliser> paulis_;
 };
 
+/**
+ * PGOp for PGOpType::MultiplexedTensoredBox, encapsulating tensor products of
+ * Ops conditioned on different values of the state of some qubits.
+ *
+ * active_paulis lists first the paulis into which the control qubits are
+ * encoded, followed by iterating over the target qubits and giving the Pauli
+ * operators mapped into +Z and +X by the conjugating circuit.
```

Comment:
Yeah, they stop the op as a whole from commuting with anything else that would act on the same space as the target qubits in the original circuit. This is a safe underestimation of commutation since we are treating the inner ops as black boxes (e.g. if all of them are secretly identity, we wouldn't spot that this multiplexor commutes with anything that is diagonal over the control qubits)